docs/guides/audio-config.md

```diff
@@ -54,8 +53,19 @@ An example to get PipeWire working on Ubuntu, which uses PulseAudio by default i
 The "Audio Volume" dialog / Audio in System Settings allow users to "Raise maximum volume", allowing to go past 100%. This
 does not offer a great deal of flexibility, it might work for getting acceptable recordings however.
 
+## Speakers
+
+Currently we have an experimental DSP config for 16,1 with 6 speakers.
+Note that each model need a special settings. Do not use it with other models as it could damage the speakers. Also do not expect same sound quality as in MacOS.
+
+[DSP16,1](https://github.com/lemmyg/asahi-audio/tree/macbookT2_16_1)
```

Comment:
Just a link to the repo doesn't explain the installation process to users, we should tell them to `git clone` it, then run the install script. It would also be simpler to tell them to run `cp` manually instead of the install script, but that might make it more confusing for new users, and it is not documented in the README.

Reply:
The readme looks like to me, except that it's missing the dependencies that are required (atleast for Ubuntu).

Reply:
> The readme looks like to me, except that it's missing the dependencies that are required (atleast for Ubuntu).

I think we should add the install steps directly on the wiki, so it is simpler to understand. A user new to this config can easily find the steps, rather than looking through another link.

Reply:
Hello,
I added the missing ubuntu deps
https://github.com/lemmyg/asahi-audio/tree/macbookT2_16_1

I believe all the required instructions are in the Readme. I didn't want to duplicate the information.

Reply:
> Hello, I added the missing ubuntu deps https://github.com/lemmyg/asahi-audio/tree/macbookT2_16_1
> 
> I believe all the required instructions are in the Readme. I didn't want to duplicate the information.

Can you close this review